docs/06-concepts/01-working-with-endpoints.md

```diff
@@ -90,3 +89,112 @@ class ExampleEndpoint extends Endpoint {
 ```
 
 The above code will not generate any server or client bindings for the example endpoint.
+
+Alternatively you can disable single methods like in the example below, where `hello` is exposed to the client, but `goodbye` is not:
+
+```dart
+import 'package:serverpod/serverpod.dart';
+
+class ExampleEndpoint extends Endpoint {
+  Future<String> hello(Session session, String name) async {
+    return 'Hello $name';
+  }
+
+  @ignoreEndpoint
+  Future<String> goodbye(Session session, String name) async {
+    return 'Bye $name';
+  }
+}
+```
+
+## Endpoint inheritance
+
+It is possible to extend existing endpoints, from your own app or other modules, by subclassing them. In case the parent endpoint was marked as `abstract` or `@ignoreEndpoint`, no client code is generated for it, but a client will be generated for your subclass – as long as it does not opt-out by any of the aforementioned ways.
+
+```dart
```

Comment:
Do you think the code example are fine to follow to see the changes?

I grouped the many variants in one, as otherwise there's too much boilerplate if hide, show, override / extend all get their own section (and there are still further variants possible and not mentioned here, like extending in a backwards compatible way by adding an optional parameter on the base class etc.).